caiman/tests/comparison_humans_online.py

```diff
@@ -101,26 +101,19 @@
     'decay_time': 0.4,
     'epochs': 3,
     'fr': 16,
-    'gSig': [8, 8],  # expected half size of neurons
+    'gSig': [8, 8],                    # expected half size of neurons
     'gnb': 2,
 }
-# % neurofinder.01.01
-params_movie[1] = {
-    'folder_name': 'N.01.01/',
-    'ds_factor': 1,
-    'fr': 8,
-    'gnb': 1,
-    'decay_time': 1.4,
-    'gSig': [6, 6]
-}
+                                       # % neurofinder.01.01
+params_movie[1] = {'folder_name': 'N.01.01/', 'ds_factor': 1, 'fr': 8, 'gnb': 1, 'decay_time': 1.4, 'gSig': [6, 6]}
```

Comment:
@pgunn 
kind of weird it chose to turn this into a single line but not the rest of the entries.

Reply:
I think the lack of a comment for this structure gave it some more flexibility. I should make how it does these consistent.

Reply:
really minor points, no need to spend time on this.